tests/scripts/setup_and_test.sh

```diff
@@ -28,3 +15,14 @@ php n98-magerun.phar config:set currency/options/base USD --root-dir $MAGENTO_DI
 php n98-magerun.phar config:set currency/options/default USD --root-dir $MAGENTO_DIR
 php n98-magerun.phar config:set currency/options/allow USD --root-dir $MAGENTO_DIR
 cp -r magento/. .
+
+mkdir ./artifacts
```

Comment:
It's generally helpful to keep the definition of this directory within the circle yml. This makes it easy to change if we change the store artifacts step

Reply:
I'll update